akd/src/append_only_zks.rs

```diff
@@ -1325,4 +1255,17 @@ mod tests {
         ));
         Ok(())
     }
+
+    fn gen_nodes(num_nodes: usize) -> Vec<Node> {
```

Comment:
Maybe this can go into `utils.rs` and be only included in test builds?

Reply:
Good suggestion! I think we may have a few other spots floating around where this is copy/pasted.